frontend/javascripts/libs/input.js

```diff
@@ -332,6 +336,8 @@ export class InputMouse {
     this.rightMouseButton = new InputMouseButton("right", 3, this, this.id);
     this.lastPosition = null;
     this.delegatedEvents = {};
+    isDragging = false;
```

Comment:
I think, this line should be removed. The variable was already initialized. Theoretically, if an InputMouse instance is constructed while another one is already active (and registered a click event), this would lead to wrong behavior :)

Reply:
Right I missed that :see_no_evil: